src/components/include/connection_handler/connection_handler.h

```diff
@@ -129,6 +129,15 @@ class ConnectionHandler {
  */
   virtual void SendEndService(uint32_t key, uint8_t service_type) = 0;
 
+  /**
+   * @brief Check is heartbeat monitoring started for specified connection key
```

Comment:
@AKalinich-Luxoft `Check...`-> `Checks if heartbeat monitoring is started...`

Reply:
@AByzhynar fixed in https://github.com/AKalinich-Luxoft/sdl_core/pull/2/commits/d3bca99da653e13fdb1e7986d4ea6f09ca37ba11